Gemfile

```diff
@@ -3,9 +3,11 @@ ruby '2.2.3'
 
 gem 'airbrake', '~> 7.1'
 gem 'devise', '~> 3.0'
+gem 'faker', '~> 1.7.0'
```

Comment:
Is this needed for all environments? It feels like it is only needed for `test` and maybe `development`.